Component/Adapter/DoctrineCacheAdapter.php

```diff
@@ -33,7 +34,14 @@ public function __construct(CacheProvider $provider)
      */
     public function getItem($key)
     {
-        return $this->provider->fetch($key);
+        if (false === $this->provider->contains($key)) {
+            return new CacheItem($key);
+        }
+
```

Comment:
The PSR proposal says, that deferred items must be returned, even in case they are not persistent. The current state of this method misses that aspect.